src/Bicep.Cli/Arguments/PublishArguments.cs

```diff
@@ -58,6 +73,8 @@ public PublishArguments(string[] args) : base(Constants.Command.Publish)
             }
         }
 
+        public string? DocumentationUrl { get; }
```

Comment:
>string [](http://example.com/codeflow?start=15&length=6)

Should we model it as a `Uri` and validate it so if the user puts in a malformed URI the command fails?

Reply:
We would be storing it as a string eventually in OciAnnotations. So I added a check to see if the provided input is a well formed uri string. Could you take a look at the changes here and let me know if that works?
https://github.com/Azure/bicep/commit/68b1ca30589890febf13f4c82e810ebdeb46a61d

Thanks!